File: docs-examples/example-versions-java/src/main/java/io/micronaut/configuration/openapi/docs/versions/UserDto.java
package io.micronaut.configuration.openapi.docs.versions;

// tag::imports[]
import jakarta.validation.constraints.NotNull;
// end::imports[]

// tag::clazz[]
public class UserDto {

    public String name;
    public int age;
    public String secondName;
    @NotNull
    public String address;
}
// end::clazz[]

File: docs-examples/example-versions-kotlin/src/main/kotlin/io/micronaut/configuration/openapi/docs/versions/UserDto.kt
package io.micronaut.configuration.openapi.docs.versions

// tag::imports[]
import jakarta.validation.constraints.NotNull
// end::imports[]

// tag::clazz[]
class UserDto {

    var name: String? = null
    var age: Int = 0
    var secondName: String? = null
    @field:NotNull
    var address: String? = null
}
// end::clazz[]
